packages/react/src/components/pickers/TagPicker/TagPicker.tsx

```diff
@@ -18,7 +18,6 @@ export class TagPickerBase extends BasePicker<ITag, ITagPickerProps> {
 
   constructor(props: ITagPickerProps) {
     super(props);
-    initializeComponentRef(this);
```

Comment:
if the only thing the constructor is doing is calling `super`, should we just take it out entirely?